etc/supervisor/app.conf.tpl

```diff
@@ -21,3 +21,15 @@ stdout_logfile_maxbytes = 0
 stderr_logfile = /dev/stderr
 stderr_logfile_maxbytes = 0
 {{- end }}
+
+{{- if (and (eq (getenv "APP") "nuxt") (eq (getenv "ENVIRONMENT") "production"))}}
+[program:nuxt]
+directory=/srv/app/
+command=nuxt-start
```

Comment:
what exactly is `nuxt-start` ? A script ? A js file ?
As far as i can see, command should be something like "node foo.js" or "npx foo"